packages/web-components/fast-foundation/src/avatar/avatar.ts

```diff
@@ -0,0 +1,78 @@
+import { attr, FASTElement } from "@microsoft/fast-element";
+
+/**
+ * An Avatar Custom HTML Element
+ * 
+ * @public
+ */
+export class Avatar extends FASTElement {
+  /**
+   * Indicates the Avatar should have a name.
+   * 
+   * @public
+   * @remarks
+   * HTML Attribute: name
+   */
+  @attr({ attribute: "name" })
```

Comment:
For the attrs, you can leave the object off here unless you need it. An example where you might use this would be:

```ts
@attr({ attribute: "background-color" })
public backgroundColor: string;
```

Reply:
I think you can drop all uses below :)